mkdocs.yml

```diff
@@ -56,28 +64,34 @@ markdown_extensions:
   - pymdownx.tilde
 
 nav:
-  - Home: index.md
-  - 'Quick Start': guide/quick-start.md
-  - 'Why Cirrus CI': why.md
-  - User Guide:
-    - 'Writing Tasks': guide/writing-tasks.md
-    - 'Linux Containers': guide/linux.md
-    - 'Windows Containers': guide/windows.md
-    - 'macOS VMs': guide/macOS.md
-    - 'FreeBSD VMs': guide/FreeBSD.md
-    - 'Docker Builder': guide/docker-builder.md
-    - 'Life of a Build': guide/build-life.md
+  - Home:
+    - Home: index.md
+  - Features:
+    - Features: features.md
```

Comment:
```suggestion
    - 'All Features': features.md
```

Reply:
This page doesn't contain All Feature but rather features from the previous Key Highlights.